### ISSUE TITLE
Add check: `homepage` should be a required field for active ontologies

### ISSUE THREAD

**Issue by cthoyt** [OBOFoundry/OBOFoundry.github.io#1668]
I think some obsolete/inactive/orphaned ontologies don't have a homepage entry, but is it the case that all new ontologies should/must have a `homepage`? The current testing suite doesn't seem to communicate this by failing a test, but rather noting it as logging deep within the build https://github.com/OBOFoundry/OBOFoundry.github.io/runs/4263986283?check_suite_focus=true#step:5:18.

The following Python script identifies EPSO (newly added in #1646) as the only non-obsolote/inactive/orphaned ontology that is missing a `homepage` entry as of 2021-11-19:

```python
import requests
import yaml
from tabulate import tabulate

OBOFOUNDRY_URL = "https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/registry/ontologies.yml"


def main():
    rows = []
    ontologies = yaml.full_load(requests.get(OBOFOUNDRY_URL).text)["ontologies"]
    headers = ["contact.name", "contact.email", "prefix", "repository"]
    for data in ontologies:
        if data.get("is_obsolete") or data.get("activity_status") in {"inactive", "orphaned"}:
            continue
        if "homepage" in data:
            continue
        prefix = data["id"]
        contact = data.get("contact", {})
        github = contact.get("github")
        rows.append((
            contact.get("label", ""),
            contact.get("email", ""),
            f"@{github}" if github else "",
            f"[`{prefix}`](http://www.obofoundry.org/ontology/{prefix})",
            data.get("repository", ""),
        ))
    rows = sorted(rows)
    print(tabulate(rows, headers=headers, tablefmt="github"))


if __name__ == '__main__':
    main()
```

**Comment by matentzn** [comment]
I think it would be a good move to make a homepage required for active ontologies - I cant really see how you can justify there not being one. @phwegner could you add a homepage for EPSO?

**Comment by cthoyt** [comment]
Closed by #1753
